include/rocksdb/db.h

```diff
@@ -893,7 +893,8 @@ class DB {
     static const std::string kCurrentSuperVersionNumber;
 
     //  "rocksdb.estimate-live-data-size" - returns an estimate of the amount of
-    //      live data in bytes.
+    //      live data in bytes. For BlobDB, it returns the exact value for live
+    //      bytes in the version.
```

Comment:
This is not entirely correct: even with BlobDB, the amount of "live" data in SST files is approximate. How about something like "For BlobDB, it also includes the exact value of live bytes in the blob files of the version." ?